packages/pages/src/dev/server/ssr/getLocalData.ts

```diff
@@ -1,24 +1,44 @@
 import path from "path";
 import fs from "fs";
 import { readdir } from "fs/promises";
+import { findTemplateModuleInternal } from "./findTemplateModuleInternal.js";
+import { ViteDevServer } from "vite";
 
 const LOCAL_DATA_PATH = "localData";
```

Comment:
Not your change, but maybe this should be part of the projectStructure? Could be a followup PR if we want to do that.

Reply:
I can add this to our cleanup doc. I'm not too familiar with ProjectStructure and this string has been here for 7 months

Reply:
Sounds good